queue/python/queue.py

```diff
@@ -0,0 +1,40 @@
+# Implementation of queue using Linked List:
+class Node:
+    def __init__(self, data):
+       self.data = data
+       self.next = None
+class Queue:
```

Comment:
E111 indentation is not a multiple of four

*Origin: PycodestyleBear (E111), Section: `all.autopep8`.*